conf/salt/project/django/run_with_db.sh

```diff
@@ -0,0 +1,5 @@
+# Shell script to set up necessary environment variables and run a PostgreSQL command
+export PGHOST="{{ pillar['env']['DB_HOST'] }}"
+export PGUSER="{{ pillar['project_name'] }}_{{ pillar['environment'] }}"
+export PGPASSWORD="{{ pillar['secrets']['DB_PASSWORD'] }}"
+exec "$@"
```

Comment:
Do I need to set PGDATABASE in order to get your approval?  Is the pull request okay as-is?